include/GafferImage/Premultiply.h

```diff
@@ -0,0 +1,83 @@
+//////////////////////////////////////////////////////////////////////////
+//
+//  Copyright (c) 2013-2015, Image Engine Design Inc. All rights reserved.
+//  Copyright (c) 2015, Nvizible Ltd. All rights reserved.
+//
+//  Redistribution and use in source and binary forms, with or without
+//  modification, are permitted provided that the following conditions are
+//  met:
+//
+//      * Redistributions of source code must retain the above
+//        copyright notice, this list of conditions and the following
+//        disclaimer.
+//
+//      * Redistributions in binary form must reproduce the above
+//        copyright notice, this list of conditions and the following
+//        disclaimer in the documentation and/or other materials provided with
+//        the distribution.
+//
+//      * Neither the name of John Haddon nor the names of
+//        any other contributors to this software may be used to endorse or
+//        promote products derived from this software without specific prior
+//        written permission.
+//
+//  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
+//  IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+//  THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+//  PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+//  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+//  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+//  PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+//  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+//  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+//  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+//  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+//////////////////////////////////////////////////////////////////////////
+
+#ifndef GAFFERIMAGE_PREMULTIPLY_H
+#define GAFFERIMAGE_PREMULTIPLY_H
+
+#include "Gaffer/StringPlug.h"
+
+#include "GafferImage/ChannelDataProcessor.h"
+
+namespace GafferImage
+{
+
+class Premultiply : public ChannelDataProcessor
+{
+
+	public :
+
+		Premultiply( const std::string &name=defaultName<Premultiply>() );
+		virtual ~Premultiply();
+
+		IE_CORE_DECLARERUNTIMETYPEDEXTENSION( GafferImage::Premultiply, PremultiplyTypeId, ChannelDataProcessor );
+
+		//! @name Plug Accessors
+		/// Returns a pointer to the node's plugs.
+		//////////////////////////////////////////////////////////////
+		//@{
+		Gaffer::StringPlug *byChannelPlug();
```

Comment:
I think the existence of the editable plug in the UI and the dropdown menu we'll add to it might make that more than clear though? I can see the argument, but for me it would be most obvious if it was called alpha, with a description of "The channel to be treated as the alpha channel, and used for multiplying with the others". I kindof think this is one of those occasions where going with the more strictly true name ends up making it less intuitive. Let's see what other folks think before changing anything.